{% extends "elixir_ita/base.html" %}

{% load staticfiles %}

{% block content %}

    <div class="table">
        <table width="910" align="center">
		<tr>
		    <td>
			<img src="{% static "img/elixir_ita_logo.jpg" %}" style="width:10%" alt="elixir_ita_logo"/>
			<img src="{% static "img/cineca_logo.jpg" %}" style="width:10%" align="right" alt="cineca_logo"/>
			<img src="{% static "img/sapienza_logo.jpg" %}" style="width:10%" align="right" alt="cineca_logo"/>
			<p align="center">ELIXIR-ITA in collaboration with CINECA and Sapienza Università di Roma</p>
			<br>
			<p align="center"> Course on: </p>
		    </td>
		</tr>		    
		<tr>
		    <td>
			<h3 align="center">{{ course.title }}</h3>
		    </td>
		</tr>
		<tr>
		    <td>
			<br>
			<p><b>&nbsp;&nbsp;&nbsp;IMPORTANT DATES</b> for this Course:
			<br />&nbsp;&nbsp;&nbsp;Deadline for applications: <b>{{ course.deadline }}</b>
			<br />&nbsp;&nbsp;&nbsp;Course date: <b> {{ course.data }} </b>
			<p><b>&nbsp;&nbsp;&nbsp;VENUE</b>:
			<br />&nbsp;&nbsp;&nbsp;{{ course.venue }}
			</p>
		    </td>
		</tr>
		<tr>
		    <td>
			<p><b>&nbsp;&nbsp;&nbsp;IMPORTANT</b>:
			<p align="justify">
			    A maximum of 20 candidates will be selected based on their research profile. Notifications of 
			    acceptance will be sent shortly after the closing date of registration.
			    <br>Priority will be given to candidates from <b>ELIXIR-ITA member institutions</b> (see the list 
			    at the bottom) and ELIXIR nodes.
			    <br><b>ELIXIR-ITA</b> will cover the expenses for the registration fee, lunches and coffee breaks. 
			    Participants are expected to pay their travel and accommodation costs.
		    </td>
		</tr>
		<tr>
		    <td>
			<p><b>&nbsp;&nbsp;&nbsp;Course description</b>:
			<font size="2">
			<p align="justify">
			    This course is aimed at PhD students and post-doctoral researchers who are applying, or will apply in the near future, high throughput sequencing technologies and the related bioinformatics tools in their research. The aim of this course is to familiarise the participants with NGS data analysis methodologies and provide hands-on training on the latest approaches. 
			    <p>Lectures will give insights into how biological knowledge can be generated from next-generation sequencing experiments. Practical training will consist of computer exercises that will enable the participants to apply the latest available tools for NGS data handling and combine them in a real workflow with downstream evolutionary analyses, under the guidance of the lecturers and teaching assistants. 
			    <p>This course will provide an introduction to next generation sequencing platforms, data analysis and tools for data quality control, including alignment to a reference sequence, data handling and visualisation, and variant calling and filtering (single nucleotide polymorphisms and structural variants). The course will be delivered using a mixture of lectures and computer based hands-on practical sessions, including group projects to be completed by the participants using the knowledge gained at the course. These will cover the general topics of population structure and admixture, demographic changes and natural selection.
			    <p>Participants with limited UNIX/Linux and R/Bioconductor experience will be provided with basic understanding of the command-line operations and the foundations of the R programming language on the first day of the course. Participants with some bioinformatics experience will be asked to help through the first day in order to create the collaborative environment needed for the group projects at the end of the course.
		    </td>
		</tr> 
		<tr>
		    <td>
			<p><b>&nbsp;&nbsp;&nbsp;Organisers</b>:
			    <p> Chiara Batini (University of Leicester, UK)
			    <br> Giovanni Chillemi (CINECA, Rome, IT)
			    <br> Allegra Via (Sapienza University, Rome, IT)
		    </td>
		</tr>
		<tr>
		    <td>
			<b>&nbsp;&nbsp;&nbsp;Instructors</b>:    
		    </td>
		</tr>	    
		<tr>
		    <td>
			<img border="0" style="margin:4px" src="{% static "img/chiara_batini_2015.jpg" %}" width="91" align="left">
			<font size="2">
			<p align="justify">
			<b>Chiara Batini</b> is an Honorary Vistor Fellow at the Department of Genetics, University of Leicester, UK. Chiara finished her PhD in 2008, carried out between Sapienza University of Rome and Universitat Pompeu Fabra (Barcelona, Spain), and followed by a long-term post-doc at University of Leicester. Her main research interests are in human evolutionary genetics and the role of culture and environment in shaping genetic diversity, with a special focus on mitochondrial and Y chromosome variation in Central African and European populations. She started working with NGS data in 2010, and since 2013 she is co-organiser and trainer for the workshops run by BBASH at University of Leicester.
			    <br><br>
			    <b>Affiliation: </b>University of Leicester, UK
			</p>
			</font>
		    </td>
		</tr>
		<tr>
		    <td>
			<img border="0" style="margin:4px" src="{% static "img/tiziana_castrignano_2015.png" %}" width="91" align="left">
			<font size="2">
			<p align="justify">
			    <b>Tiziana Castrignan&ograve;</b> received her PhD in Biophysics in 1999 from the Sapienza University (Rome, IT).
			    Her primary activity is the building and management of high performance computing services developed 
			    in many national and international bioinformatics research projects. Since 2000 she works in the Computational 
			    Chemistry Group at CASPUR (now CINECA), Rome, where she collaborates with the scientists in the field of 
			    bioinformatics. Today she works in the CINECA-SCAI user support group. 
			    Since 2008 she acts as external professor of Computational Biology in the Department of Informatics at the 
			    Sapienza University of Rome. 
			    <br><br>
			    <b>Affiliation: </b>CINECA, Roma, IT
			</p>
			</font>
		    </td>
		</tr>
		<tr>
		    <td>
			<img border="0" style="margin:4px" src="{% static "img/giovanni_chillemi_2015.jpg" %}" width="91" align="left">
			<font size="2">
			<p align="justify">
			    <b>Giovanni Chillemi</b> holds a Master Degree in Chemical Engineering from the Sapienza University 
			    (Rome, IT) and a PhD in Biochemistry and Molecular Biology from the University of Rome "Tor Vergata". 
			    From 1997 to 2013 he worked at CASPUR, the computing centre based in Rome (now CINECA) in the 
			    computational chemistry group, collaborating with the scientists in parallelising and enabling applications 
			    for computational chemistry, structural biology and computational genomics. Since 2014 he works in the 
			    CINECA SCAI user support group, collaborating with scientists in enabling HPC applications. He acted as 
			    external professor at the Department of Biology, University of Rome “Tor Vergata” and at the Department 
			    of Pharmaceutical Sciences, University of Bologna. In 2014 he obtained the Italian national scientific 
			    qualification as Associate Professor in "Molecular Biology" and "General and Clinical Biochemistry". 
    			    <br><br>
			    <b>Affiliation: </b>CINECA, Roma, IT
			</p>
			</font>
		    </td>
		</tr>
		<tr>
		    <td>
			<img border="0" style="margin:4px" src="{% static "img/vincenza_colonna_2015.png" %}" width="91" align="left">
			<font size="2">
			<p align="justify">
			    <b>Vincenza Colonna</b> is group leader at the Institue of Genetics and Biophysics of the Italian National Research Council in Napoli, Italy. She graduated from University of Napoli Federico II (Italy) and did postdoctoral work  at University of Ferrara (Italy) and at the Wellcome Trust Sanger Institute (UK). She was lectures in Genetics and Bioinformatics at the University of Ferrara (Italy). She works in human genomics and her main research interests are natural selection and consanguineous populations. Her research focuses on causes and consequences of genetic diversity and natural selection.  She is also leading a project on training in Bioinformatics  (http://www.igb.cnr.it/obilab). 			
			    <br><br>
			    <b>Affiliation: </b>CNR (National Research Council), Napoli, IT
			</p>
			</font>
		    </td>
		</tr>
		<tr>
		    <td>
			<img border="0" style="margin:4px" src="{% static "img/pille_hallast_2015.jpg" %}" width="91" align="left">
			<font size="2">
			<p align="justify">
			    <b>Pille Hallast</b> works as senior scientist in human molecular genetics at the Institute of Molecular and Cell Biology, University of Tartu, Estonia. Pille started working in Tartu in September 2015, after finishing a 5-years postdoc with Prof. Mark Jobling at University of Leicester, United Kingdom. Before that, she did her PhD in molecular diagnostics in University of Tartu. Her main research interests are in understanding the molecular dynamics of structurally complex genomic regions, and the genetic diversity of Y chromosome and its use in population genetics. Her first contact with NGS data generation, handling and analysis was during her postdoc without any prior training in bioinformatics. 
			    <br><br>
			    <b>Affiliation: </b>University of Tartu, Estonia
			</p>
			</font>
		    </td>
		</tr>
		<tr>
		    <td>
			<img border="0" style="margin:4px" src="{% static "img/allegra_2015.jpg" %}" width="91" align="left">
			<font size="2">
			<p align="justify">
			    <b>Allegra Via</b> is a physicist and scientific researcher in the Biocomputing Unit of Sapienza
			    Universit&agrave; (Roma, IT) where she also teaches Macromolecular Structure, Python programming, 
			    Bioinformatics and Biochemistry. Her main research interests include protein structural 
			    bioinformatics, protein function prediction and analysis, and protein interactions. On top of her research
			    activities, she is strongly involved in the design, organisation and delivery of bioinformatics training courses. 
			    She is the Training Coordinator for <b>ELIXIR Italy</b>, 
			    the <b>ELIXIR.ITA</b> local technical coordinator for Sapienza Universit&agrave;, and a member of the Global 
			    Organisation for Bioinformatics Learning, Education and Training, <b>GOBLET</b>. 
			    <br><br>
			    <b>Affiliation: </b>Sapienza Universit&agrave;, Roma, IT
			</p>
			</font>
		    </td>
		</tr>
		<tr>
		    <td>
			<p align="justify">
			<b>Target audience</b>: This course is aimed at wet-lab biologists who are embarking upon research projects that will require the handling and analysis of next generation sequencing data in organisms for which a reference genome is available.
			</p>
		    </td>	
		</tr>
		<tr>
		    <td>
			<p align="justify">
			<b>Prerequisites</b>: A significant proportion of the course will be computer-based using command line tools in the Unix environment, therefore, in order to gain maximum benefit from the course all attendees will be required to have basic Unix skills. This means applicants must at a minimum be able to move easily around the file sytem and make directories (e.g. pwd, cd, ls, mkdir) and manage files (e.g. cp, mv, rm).
		    </td>	
		</tr>
		<tr>
		    <td>
			<p align="justify">
			<b>Learning objectives</b>: Participants in the course will gain familiarity with NGS methodologies and their different applications when a reference genome is available. These will cover from alignment to variant calling of both single nucleotide polymorphisms and structural variants. Participants will apply the knowledge gained during the course into a real group project on the last day.
		    </td>	
		</tr>
		<tr>
		    <td>
			<p align="justify">
			<b>Cancellation policy</b>:  Attendance is limited to 20 participants. We expect many more applications. 
			Accepted participants commit to attend the course for its whole duration. Failure to attend training sessions 
			is disruptive. Moreover, it blocks other candidates from participating. Therefore a cancellation policy is in
			place so that only written requests presented two weeks in advance relatively to the course starting date
			are accepted. Otherwise, the candidate is short-listed and will not be accepted if and when applying for 
			the next ELIXIR-ITA course on the same subject. 
		    </td>	
		</tr>
		<tr>
		    <td>
			<div class="course">	
			    <h4><a href="{% url "programme" pk=course.pk %}">Course detailed programme</a></h4>
			    <h4><a href="{% url "application" pk=course.pk %}">Application</a></h4>
			</div> 
		    </td>	
		</tr>
		<tr>
		    <td>
			<font size="4">
			<p><b>ELIXIR-ITA member institutions</b></p>
			</font>
			<p>CNR (ELIXIR-ITA coordinator)
			<br>CRS4
			<br>CINECA
			<br>INFN
			<br>GARR
			<br>Sapienza Università di Roma
			<br>Università di Bari
			<br>Università di Bologna
			<br>Università di Milano
			<br>Università di Milano Bicocca
			<br>Università di Padova
			<br>Università di Roma "Tor Vergata"
			<br>Università della Tuscia
			
		    </td>	
		</tr>
	</table>
    </div>


{% endblock content %}
